native/jni/core/magiskinit.c

```diff
@@ -428,11 +428,24 @@ int main(int argc, char *argv[]) {
 	if (cmd.skip_initramfs || access("/sepolicy", R_OK) != 0) {
 		char partname[32];
 		struct device dev;
+		char vendor[10];
+		char vendorpath[128];
+		int fd;
 
 		// Mount sysfs
 		mkdir("/sys", 0755);
 		xmount("sysfs", "/sys", "sysfs", 0, NULL);
 
+		// Check if device uses early-mount
+		if (access("/sys/firmware/devicetree/base/firmware/android/fstab/vendor/dev", R_OK) == 0) {
+			fd = open("/sys/firmware/devicetree/base/firmware/android/fstab/vendor/dev", O_RDONLY | O_CLOEXEC);
+			vendorpath[read(fd, vendorpath, sizeof(vendorpath))] = '\0';
+			strcpy(vendor, strrchr(vendorpath, '/') + 1);
+			close(fd);
+		} else
```

Comment:
Where's the `}`?

Reply:
@TheScarastic ,
So that's the reason you said LOL ?????